tests/unit/test_post_schema.py

```diff
@@ -0,0 +1,18 @@
+from unittest import TestCase
+
+import pytest
+from fastapi import status
+
+
+class PostSchemaTest(TestCase):
+    @pytest.fixture(autouse=True)
+    def prepare_fixture(self, test_client):
+        self.test_client = test_client
+
+    def test_post_schema_with_valid_json(self):
+        response = self.test_client.post(
+            "/convert/schema?current_version=1&target_version=2",
+            json={"valid_json": "valid_json"},
+        )
+        assert response.status_code == status.HTTP_200_OK
+        assert response.json()["valid_json"] == "valid_json"
```

Comment:
This could live in `tests/unit/test_schema_router.py` to follow the same structure and layout as the implementation I think?

Also, the test could be simplified to pass in the `test_client` fixture as an argument like: 

```suggestion
def test_post_schema_with_valid_json(test_client):
    response = test_client.post(
        "/convert/schema?current_version=1&target_version=2",
        json={"valid_json": "valid_json"},
    )
    assert response.status_code == status.HTTP_200_OK
    assert response.json()["valid_json"] == "valid_json"
```

Reply:
Good point, simplified the test.